tests/python_tests/test_vlm_pipeline.py

```diff
@@ -269,34 +269,40 @@ def test_sampling(config, cache):
     pipe.generate(prompts[0], image=image, generation_config=config)
 
 
-@pytest.mark.skip(reason="CVS-160580: tests/python_tests/test_vlm_pipeline.py::test_perf_metrics fails")
 @pytest.mark.precommit
 @pytest.mark.nightly
 def test_perf_metrics(cache):
     import numpy as np
+    from time import perf_counter_ns
     models_path = get_ov_model("katuni4ka/tiny-random-minicpmv-2_6", cache)
 
     images = [get_image_by_link(image_links[0])]
+    max_new_tokens = 30
 
+    start_time = perf_counter_ns()
     pipe = VLMPipeline(models_path, "CPU")
-    result = pipe.generate(prompts[0], images=images, generation_config=GenerationConfig(max_new_tokens=30))
+    start_generate = perf_counter_ns()
+    load_time = (start_generate - start_time) / 1000000
+
+    result = pipe.generate(prompts[0], images=images, generation_config=GenerationConfig(max_new_tokens=max_new_tokens))
+    generate_time = (perf_counter_ns() - start_generate) / 1000000
 
     perf_metrics = result.perf_metrics
 
     assert perf_metrics is not None
 
-    assert 0 < perf_metrics.get_load_time() < 2000
-    assert 0 < perf_metrics.get_num_generated_tokens() < 100
-    assert 0 < perf_metrics.get_num_input_tokens() < 100
-    assert 0 < perf_metrics.get_ttft().mean < 1000
-    assert 0 < perf_metrics.get_tpot().mean < 100
-    assert 0 < perf_metrics.get_ipot().mean < 100
-    assert 0 < perf_metrics.get_throughput().mean < 1000
-    assert 0 < perf_metrics.get_inference_duration().mean < 1000
-    assert 0 < perf_metrics.get_generate_duration().mean < 1000
-    assert 0 < perf_metrics.get_tokenization_duration().mean < 100
-    assert 0 < perf_metrics.get_detokenization_duration().mean < 10
-    assert 0 < perf_metrics.get_prepare_embeddings_duration().mean < 100
+    assert 0 < perf_metrics.get_load_time() < load_time
+    assert 0 < perf_metrics.get_num_generated_tokens() < generate_time
+    assert 0 < perf_metrics.get_num_input_tokens() < generate_time
+    assert 0 < perf_metrics.get_ttft().mean < generate_time
+    assert 0 < perf_metrics.get_tpot().mean < generate_time
+    assert 0 < perf_metrics.get_ipot().mean < generate_time
+    assert 0 < perf_metrics.get_throughput().mean < max_new_tokens / ((generate_time - perf_metrics.get_ttft().mean) / 1000)
+    assert 0 < perf_metrics.get_inference_duration().mean < generate_time
+    assert 0 < perf_metrics.get_generate_duration().mean < generate_time
+    assert 0 < perf_metrics.get_tokenization_duration().mean < generate_time
+    assert 0 < perf_metrics.get_detokenization_duration().mean < generate_time
+    assert 0 < perf_metrics.get_prepare_embeddings_duration().mean < generate_time
```

Comment:
Can you do the same asserts with `std` members? `generate_time*generate_time` would be the upper bound. Not sure if it can be `(generate_time/2)^2`.

Reply:
Done.